index.js

```diff
@@ -57,6 +58,37 @@ module.exports = {
     // called and that is dynamic, so we need to make sure we are requiring the correct parent file everytime.
     delete require.cache[__filename];
 
+    // Finally return the target file's config.
+    return require(targetFile);
+  },
```

Comment:
Wasn't this already present ? Why showing up as newly added 😕

Reply:
GitHub diff is showing lines 92 and below as existing.